### PR TITLE
Move export and filter buttons to the right above the table

### PR DESCRIPTION
**Pull request by slav-petkov-saorsa**
Background:
https://trello.com/c/jUyj4aiB/436-move-export-and-filter-buttons-to-the-right-above-the-table

Solution:
- Enabled wrap on the flexbox so the filters can go on the next line on narrower screens

### CODE REVIEW

File: app/assets/stylesheets/bulk-actions.scss [hunk 1]
```diff
@@ -4,14 +4,21 @@
     display: flex;
     flex-direction: row;
     flex: 1 0 auto;
+    flex-wrap: wrap;
     align-items: center;
     .last-element {
       margin-left: auto;
       display: flex;
-      flex-direction: row-reverse;
+      //flex-direction: row-reverse;
```
Comment (jamiebuckley) [inline app/assets/stylesheets/bulk-actions.scss:12]: Can we remove the commented code?

**Comment by jamiebuckley** [comment]
I think it's slightly confusing too that the filters/close filters buttons appear on the right, but the clear filters appears on the left
Screenshot from 2020-04-30 10-34-05

**Comment by slav-petkov-saorsa** [comment]
> I think it's slightly confusing too that the filters/close filters buttons appear on the right, but the clear filters appears on the left
> Screenshot from 2020-04-30 10-34-05

You are right. That's an issue I need to fix. It happens when clicking on another link. If you close the filters and open again, both buttons will be on the right hand side. Discovered it this morning

File: app/assets/stylesheets/bulk-actions.scss (hunk 1 above)
Comment (slav-petkov-saorsa) [reply app/assets/stylesheets/bulk-actions.scss:12]: Will do after I finish the assessment dropdown.

**Comment by timwis** [comment]
Now that my PR has merged, it's created a conflict here, srrrrrryyyyy 😬

**Pull request merged**